openrpc.json

```diff
@@ -511,6 +511,36 @@
         }
       ]
     },
+    {
+      "name": "wallet_scanQRCode",
+      "tags": [
+        {
+          "$ref": "#/components/tags/MetaMask"
```

Comment:
I think we are using `MetaMask` as a tag for extension and `Mobile` for mobile. we should think through the tags here